ledger/accountdb.go

```diff
@@ -735,6 +762,12 @@ func encodedAccountsRange(tx *sql.Tx, startAccountIndex, accountCount int) (bals
 	}
 
 	err = rows.Err()
+	if err == nil {
+		// the encodedAccountsRange typically called in a loop iterating over all the accounts. This could clearly take more than the
+		// "standard" 1 second, so we want to extend the timeout on each iteration. If the last iteration takes more than a second, then
+		// it should be noted. The one second here is quite liberal to ensure symmetrical behaviour on low-power devices.
+		db.ResetTransactionWarnDeadline(ctx, tx, time.Now().Add(time.Second))
```

Comment:
maybe indicate that the returned error value is intentionally ignored here (and below)?